pkg/minikube/cluster/cluster.go

```diff
@@ -35,7 +36,7 @@ import (
 )
 
 const (
-	remotePath = "/var/lib/localkube/certs"
+	remotePath = "/var/lib/localkube/certs/"
```

Comment:
Rename to `remoteCertPath`.
Is the `/var/lib/localkube/` path already a constant on the minikube side, or may one change that?